File: challenge4.js

const registerToVote = (name, unregisteredVoters) => {
  // Code here!
  let names = []
  for (let ii = 0; ii < unregisteredVoters.length; ii++){
    if (unregisteredVoters[ii] !== name){
      names.push(unregisteredVoters[ii])
    }
  }
  return names
}

File: ruby/challenge4.rb
def registerToVote (name, unregisteredVoters)
  names = []
  for x in unregisteredVoters
    if (x != name)
        names << x
    end
  end
  return names
end

# name = 'Bradley'
# unregisteredVoters = ['Jake', 'Alanna', 'Bradley', 'Stephanie']
# registerToVote(name, unregisteredVoters)
